client/src/containers/Thread/Thread.js

```diff
@@ -1,5 +1,5 @@
-import React from "react";
-import { 
+import React, { useEffect } from "react";
```

Comment:
Please change everything that is named `Thread`* to `Message`* or `Workspace`* so that it is better aligned with the model of our project.

Reply:
@nimish-gupta, any thoughts on this, because inconsistencies with naming will mess with our understanding of what the component should do?

Reply:
Basic entities like messages, thread, workspaces should be defined before starting the project. And then stick to the same conventions for defining the component.

Reply:
Yes, the above-mentioned convention was what we decided before starting the project.

Reply:
I beg to differ here. I remember discussing it at some point, but nothing was decided about it. This happens because a lot of one to one discussions are happening and being considered it as decided. 

If you wish to rename it, please do. Just make sure it doesn't break anything else.

![image](https://user-images.githubusercontent.com/1090611/61477230-64ac6700-a9ac-11e9-8890-fe9685a6c1df.png)

![image](https://user-images.githubusercontent.com/1090611/61477290-87d71680-a9ac-11e9-82ec-07b1aaac7edd.png)

![image](https://user-images.githubusercontent.com/1090611/61477321-96bdc900-a9ac-11e9-941b-29369cd4b0d2.png)